universal-application-tool-0.0.1/app/views/loginForm.scala.html

```diff
@@ -0,0 +1,10 @@
+@(url: String)(implicit request: Http.Request)
+<form action="@url?client_name=FormClient" method="POST">
+    <input type="text" name="username" value="" />
+    <p />
+    <input type="password" name="password" value="" />
+    <p />
+    @helper.CSRF.formField
+
+    <input type="submit" name="submit" value="Submit" />
+</form>
```

Comment:
Suggestion (non-blocking): we're dropping the scala Twirl template system in favor of [J2HTML](http://j2html.com/) (see demo code in our [README](https://github.com/seattle-uat/universal-application-tool#views)), please rewrite this form using J2HTML or file an issue to do so.

Reply:
Done!